crate/kmip/src/openssl/private_key.rs

```diff
@@ -176,11 +176,23 @@ fn ec_private_key_from_scalar(
     curve: &RecommendedCurve,
 ) -> Result<PKey<Private>, KmipError> {
     let (nid, privkey_size) = match curve {
+        // P-CURVES
         RecommendedCurve::P256 => (Nid::X9_62_PRIME256V1, 32),
         RecommendedCurve::P192 => (Nid::X9_62_PRIME192V1, 24),
         RecommendedCurve::P224 => (Nid::SECP224R1, 28),
         RecommendedCurve::P384 => (Nid::SECP384R1, 48),
         RecommendedCurve::P521 => (Nid::SECP521R1, 66),
+        // K-CURVES
```

Comment:
Do we want to support curves others than P-curves ?
I know with certainty that Boolean curves are deprecated.

Reply:
Update: NIST.SP.800-186 Section 3.1.2 Table 2. specifically says that K/B curves are deprecated for any usage (signature, key agreement).

Reply:
done in 4d5a656a2044162d9356e159bc8723c274db3497